cloud-bigdata.rst

```diff
@@ -0,0 +1,105 @@
+
+Overview
+========
+
+This project defines the `Rackspace Cloud Big Data <http://www.rackspace.com/en-us/cloud/big-data>`__ (CBD) team's `Cloud Orchestration <http://www.rackspace.com/en-us/cloud/orchestration>`__ (`OpenStack Heat <https://wiki.openstack.org/wiki/Heat>`__) Resource Plugin. The CBD plugin enables accessing Rackspace Cloud Big Data services through `Heat Orchestration Templates <http://docs.openstack.org/developer/heat/template_guide/hot_guide.html>`__ (HOTs) by editing a YAML file or through the `Rackspace Control Panel <https://mycloud.rackspace.com/>`__ Orchestration management interface. The Rackspace Cloud Big Data Heat resource plugin identifier is:
+
+**Rackspace::Cloud::BigData**
+
+The CBD plugin may be used for creating and deleting CBD services.
+
+Example Template
+================
+
+Start by adding the top-level template sections:
+
+.. code:: yaml
+
+    heat_template_version: 2015-10-15
+
+    description: |
+      This is a Heat template to deploy a Rackspace::Cloud::BigData cluster.
+
+    resources:
+
+    outputs:
+
+Resources Section
+-----------------
+Add a Rackspace::Cloud::BigData resource with required values as shown by this example configuration:
+
+.. code:: yaml
+
+      cbd_cluster:
+        type: "Rackspace::Cloud::BigData"
+        properties:
+          stackId:       HADOOP_HDP2_2
+          clusterName:   my_cbd_cluster
+          clusterLogin:  my_ssh_login
+          flavor:        Small Hadoop Instance
+          numSlaveNodes: 3
+          publicKeyName: my_ssh_pub_key_name
+          publicKey:     "... paste public SSH key here ..."
+
+
+Outputs Section
+---------------
+
+Add the following to your outputs section:
+
+.. code:: yaml
+
+      cluster_id:
+        value: { get_resource: cbd_cluster }
+        description: Cloud Big Data Cluster ID
+
+      cbd_version:
+        value: { get_attr: [cbd_cluster, cbdVersion] }
+        description: Cloud Big Data version
+
+
+Full Template
+-------------
+
+.. code:: yaml
+
+    heat_template_version: 2015-10-15
+
+    description: |
+      This is a Heat template to deploy a Rackspace::Cloud::BigData cluster.
+
+    resources:
+      cbd_cluster:
+        type: "Rackspace::Cloud::BigData"
+        properties:
+          stackId:       HADOOP_HDP2_2
+          clusterName:   my_cbd_cluster
+          clusterLogin:  my_ssh_login
+          flavor:        Small Hadoop Instance
+          numSlaveNodes: 3
+          publicKeyName: my_ssh_pub_key_name
+          publicKey:     "... paste public SSH key here ..."
+
+    outputs:
+      cluster_id:
+        value: { get_resource: cbd_cluster }
+        description: Cloud Big Data Cluster ID
+
+      cbd_version:
+        value: { get_attr: [cbd_cluster, cbdVersion] }
+        description: Cloud Big Data version
+
+Reference Documentation
+=======================
+
+- `Rackspace Cloud Big Data Developer Guide, v1 <http://docs.rackspace.com/cbd/api/v1.0/cbd-devguide/cbd-devguide-20150423.pdf>`__
```

Comment:
Can you add [v2](http://docs.rackspace.com/cbd/api/v1.0/cbd-devguide-2/cbd-devguide-2-20150630.pdf) guide instead.